dev/integration_tests/gradle_deprecated_settings/android/project-camera_android.lockfile

```diff
@@ -1,157 +0,0 @@
-# This is a Gradle generated file for dependency locking.
-# Manual edits can break the build and are not advised.
```

Comment:
Do we not need a lockfile for this test app?

Reply:
This lockfile was replaced by the one for `camera_android_camerax`
https://github.com/flutter/flutter/blob/master/dev/integration_tests/gradle_deprecated_settings/android/project-camera_android_camerax.lockfile

Reply:
ahh, this is per dep, thanks!